docs/DeveloperGuide.md

```diff
@@ -1,8 +1,42 @@
 ---
 layout: page
-title: Travelling BusinessMan, TBM Developer Guide
+title: Travelling BusinessMan(TBM) Developer Guide
 ---
 
+
+## Table of Contents
+
+- [Table of Contents](#table-of-contents)
+- [**Introduction**](#introduction)
+  - [Purpose](#purpose)
+  - [Scope](#scope)
+- [**Setting up, getting started**](#setting-up-getting-started)
+- [**Design**](#design)
+  - [Architecture](#architecture)
+  - [UI component](#ui-component)
+  - [Logic component](#logic-component)
+  - [Model component](#model-component)
+  - [Storage component](#storage-component)
+  - [Common classes](#common-classes)
+- [**Implementation**](#implementation)
+  - [\[Proposed\] Undo/redo feature](#proposed-undoredo-feature)
+    - [Proposed Implementation](#proposed-implementation)
+    - [Design consideration:](#design-consideration)
+      - [Aspect: How undo & redo executes](#aspect-how-undo--redo-executes)
+  - [\[Proposed\] Data archiving](#proposed-data-archiving)
+- [**Documentation, logging, testing, configuration, dev-ops**](#documentation-logging-testing-configuration-dev-ops)
+- [**Appendix: Requirements**](#appendix-requirements)
+  - [Product scope](#product-scope)
+  - [User stories](#user-stories)
+  - [Use cases](#use-cases)
+  - [Non-Functional Requirements](#non-functional-requirements)
+  - [Glossary](#glossary)
+- [**Appendix: Instructions for manual testing**](#appendix-instructions-for-manual-testing)
+  - [Launch and shutdown](#launch-and-shutdown)
+  - [Deleting a person](#deleting-a-person)
+  - [Saving data](#saving-data)
```

Comment:
The table of contents is actually generated by GitHub with 
```
* Table of Contents
{:toc}
```

With this addition we will have two copies of the TOC on our documentation website.

![image](https://user-images.githubusercontent.com/1027621/94670390-14b58800-0345-11eb-948d-b7bb0886d085.png)

![image](https://user-images.githubusercontent.com/1027621/94670422-1d0dc300-0345-11eb-8c98-4ab3685b0509.png)

While editing this, could you also change the spelling of "Travelling Businessman" in "Glossary" to have a capitalized "M"?

Reply:
done